### DISCUSSION TITLE
Welcome to ThisIsWin11 Discussions!

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/ThisIsWin11#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ExaFlop1** [comment]
Please, add Windows Subsystem for Android.

**Reply by TheSmartBB8** [reply]
DO NOT DOWNLOAD IT CAUSES 100% CPU USAGE AND BRICKS SYTEM

**Comment by noobj2** [comment]
Can this app bring back full screen start menu to windows 11?

**Comment by sunnyjcb** [comment]
please add disable windows defender permanently

**Comment by bobalazs** [comment]
When i uninstalled all system apps with pumpedapp, any way to manually restore them? with a windows command perhaps?
I emptied bin unfortuntly so i don't have any installed system apps, and no not restoreable.

**Comment by lornjohn** [comment]
I need help, I download ThisisWin11 yesterday and this morning my laptop screen is black and I can't do anything. How do I revert to pre This is Win11?

**Reply by lornjohn** [reply]
Thanks for the reply, if I could see the screen to see what I was doing, I would do a system restore.

**Comment by kelltom** [comment]
Any plans to add a feature that disables unnecessary services?

**Comment by gionny74** [comment]
cortana non si disistalla non appare piu nella lista app su thisiswin11 ma nella lista app risulta sempre come risolvere????

**Comment by Eta-Beta** [comment]
Hello, to communicate I use google translator so I apologize if it is not very clear what I write. A huge thank you to the author (s) of ThisIsWin11 is done really well. I am an old user of TweakUI, but to say user is too much because not knowing English then it was really hard to use it. Luckily today we have excellent translation tools and we have an advantage but if it were in Italian it would be better for me. I downloaded the source code hoping that there was a single English language file to translate like in other software (I'm not a programmer) but it isn't. I would like to translate this wonderful software and with google translator I can do it but I need to understand how to insert the translations, so if anyone can help me I would be grateful.

ITALIANO:
Buongiorno, per comunicare uso google traduttore per cui mi scuso se non è molto chiaro quello che scrivo. Un grandissimo ringraziamento per l'autore/i di ThisIsWin11 è fatto veramente bene . Sono un vecchio utilizzatore di TweakUI , dire utilizzatore però è troppo perché non conoscendo l'inglese allora era veramente dura utilizzarla. Oggi per fortuna abbiamo ottimi strumenti di traduzione e siamo avvantaggiati ma se fosse in italiano sarebbe meglio per me. Ho scaricato il codice sorgente sperando che ci fosse un unico file di lingua inglese da tradurre come in altri software (non sono un programmatore) ma non è cosi. Vorrei tradurre questo stupendo software e con google traduttore lo posso fare ma avrei bisogno di capire come inserire le traduzioni, per cui se qualcuno può aiutarmi gliene sarei grato.
